packages/@vue/cli-plugin-babel/index.js

```diff
@@ -44,8 +46,17 @@ module.exports = (api, {
           .loader('thread-loader')
     }
 
-    jsRule
+    const babelLoader = jsRule
       .use('babel-loader')
         .loader('babel-loader')
+
+    // use project's babel config for dependencies
+    if (transpileDependencies.length !== 0) {
+      // `babelrc` = false
+      const config = babelHelpers.loadUsersConfig(api.resolve('.'))
+
+      babelLoader
+        .options(config)
```

Comment:
How does `babelLoader` here end up calling Babel exactly? I'd normally expect the loader to do all the work of reading the config, so I'm not totally following why this tries to read the config up front.